### ISSUE TITLE
Nation Zone issue/Console error

### ISSUE THREAD

**Issue by Momshroom** [TownyAdvanced/Towny#2985]
This ticket template is for bug/error reports. If you are suggesting a feature and not reporting a bug please delete all the text here and begin your ticket Title with Suggestion:

### What steps will reproduce the problem?
1. set up towny to use nation zones and set it to not be only capitals
2. go to a town not in your nation and go just outside
3. break a block

### What is the expected output?
1. it's expected that you won't be able to, but you can.
2. You get a console error (listed below)
3.

### What is your Towny version?
Do not say "latest".
0.92.0.12

### What is your spigot/craftbukkit version?
Do not say "latest".
CraftBukkit version git-Spigot-eb3d921-b9f1615 (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT)

### Please use Pastebin.com to link the following files:

**Skipping these files is not an option. It is the easiest way to diagnose an issue ticket properly. If you do skip this step then your ticket will most likely be deleted and you will be asked to resubmit.**
1. Your full server startup from the logs\latest.log : https://pastebin.com/qsES3FyN
2. Your Towny config.yml (if using MySQL please remove password,) : https://pastebin.com/qJzfvdNi
3. Your townyperms.yml :https://pastebin.com/FWeCnHPX
4. Your log's error : https://pastebin.com/Wv1cuPJP

(optional) If this is to do with permissions, 
5. Your permissions file : 

(optional) If this is to do with chat,
6. Your chatconfig.yml :
7. Your channels.yml :

**Comment by Momshroom** [comment]
Followup information:
If I disable nation zones in the config and attempt to break in the same location, I no longer get a console error.

**Comment by LlmDl** [comment]
I'll do some testing, there's nothing immediately obvious. Which wilderness permission nodes are you giving players in luckperms?

**Comment by LlmDl** [comment]
The console error you have doesn't appear to be the same issue. It has to be occurring inside of towns and not anything to do with the wilderness or NationZones.

**Comment by Momshroom** [comment]
Thank you.  

members get: (yes, I know it's redundant)
- towny.wild.*
- towny.wild.build.103
- towny.wild.build.111
- towny.wild.build.115
- towny.wild.build.17
- towny.wild.build.6
- towny.wild.build.83
- towny.wild.build.86
- towny.wild.destroy.103
- towny.wild.destroy.106
- towny.wild.destroy.111
- towny.wild.destroy.115
- towny.wild.destroy.14
- towny.wild.destroy.15
- towny.wild.destroy.16
- towny.wild.destroy.17
- towny.wild.destroy.18
- towny.wild.destroy.21
- towny.wild.destroy.31
- towny.wild.destroy.37
- towny.wild.destroy.38
- towny.wild.destroy.39
- towny.wild.destroy.40
- towny.wild.destroy.50
- towny.wild.destroy.56
- towny.wild.destroy.6
- towny.wild.destroy.73
- towny.wild.destroy.74
- towny.wild.destroy.78
- towny.wild.destroy.81
- towny.wild.destroy.82
- towny.wild.destroy.83
- towny.wild.destroy.86
- towny.wild.switch.64

**Comment by LlmDl** [comment]
This has been fixed as of 0.92.0.13:
```
- Fix NationZones, which I broke back in a commit that did the badtouch to the playercacheutil.
```
